hydromt/raster.py

```diff
@@ -26,7 +22,7 @@
 import pyproj
 import rasterio.fill
 import rasterio.warp
-import rioxarray
+import rioxarray  # noqa: F401
```

Comment:
I'm guessing the # noqa: F401 and others are used by some of the pre-commit tools? Then good to add the most common ones and explanation in the developers docs

Reply:
indeed. F401 means no unused imports. I'll add them to the docs.